client/src/views/asset-list.js

```diff
@@ -1,12 +1,37 @@
 import Snabbdom from 'snabbdom-pragma'
-
+import { formatAssetAmount } from './util'
 import layout from './layout'
 
-export default ({ assetMap, t, ...S }) => {
+const staticRoot = process.env.STATIC_ROOT || ''
+export default ({ assetMap, assetList, loading, t, ...S }) => {
```

Comment:
`assetMap` isn't used here, if you don't unpack it to a variable here it'll remain part of `S` and then you won't have to specify it again below